src/modules/todo/constants/index.ts

```diff
@@ -0,0 +1 @@
+export const FETCH_TODO_LIST_QUERY_KEY = 'todos'
```

Comment:
It's better to put this const inside hook itself, as in this case it's internal const that belong to hook only not for todo module.